docs/01.GETTING-STARTED.md

```diff
@@ -4,64 +4,70 @@ Currently, only Ubuntu 14.04+ is officially supported as primary development env
 
 There are several dependencies, that should be installed manually. The following list is required for building:
 
-- `gcc` higher than `4.8.2`
+- `gcc` >= `4.8.2`
   - native
   - arm-none-eabi
-- `cmake` higher than `2.8.12.2`
-- `make` higher than `3.81`
-- `bash` higher than `4.3.11`
-- `cppcheck` higher than 1.61
-- `vera++` higher than 1.2.1
+- `cmake` >= `2.8.12.2`
+- `make` >= `3.81`
```

Comment:
Technically we don't need make anymore, do we? E.g. a CMake build with the Ninja generator should work just fine without even having make installed.